graphql/schema/schemagen.go

```diff
@@ -669,6 +673,13 @@ func genDgSchema(gqlSch *ast.Schema, definitions []string,
 						}
 					}
 
+					embedding := f.Directives.ForName(embeddingDirective)
+					if embedding != nil {
+						// embeddingValidation ensured GQL type is [Float]
```

Comment:
What happens if we first have a schema with normal [Float], but then we add embedding